docs/source/variables.ipynb

```diff
@@ -160,6 +160,79 @@
     "from featuretools.variable_types import list_variable_types\n",
     "list_variable_types()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining Custom Variable Types"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Users can define their own variables types. For example, to make a custom variable type called Age, do the following:"
```

Comment:
> Users can define their own variables types.

should be variable types

> For example, to make a custom variable type called Age, do the following:

maybe "run the following code:"

Reply:
Fixed:
https://github.com/FeatureLabs/featuretools/pull/1066/commits/601a412e6e8abe38cf357506fa5055bd3fe1a8c0#diff-788cca9c6a491b8eed11068ab47ed94cR175